src/dlgTriggerEditor.cpp

```diff
@@ -6313,6 +6324,20 @@ void dlgTriggerEditor::saveOpenChanges()
     };
 }
 
+void dlgTriggerEditor::timerEvent(QTimerEvent *event)
+{
+    Q_UNUSED(event);
+
+    if (isActiveWindow()) {
+        autoSave();
+    }
+}
+
+void dlgTriggerEditor::autoSave()
+{
+    mpHost->saveProfile(nullptr, QStringLiteral("autosave"));
```

Comment:
You are handing a `nullptr` here, however, `saveProfile` runs `isEmpty()` on it. Is that allowed?

Reply:
It seems to work :thinking: I'll change it to a blank QString though.

Reply:
Might be that by providing `nullptr`, the default value of this optional parameter will be used :thinking: But I can't find an example for this at the moment.